### ISSUE TITLE
Improve share link for Facebook

### ISSUE THREAD

**Issue by chrisbolum** [selbekk/reecap#77]
When we (and others hopefully) post link to a Reecap event on Facebook, the link looks very neutral - see attached image on Slack. Is it possible to adjust how it will look when shared? I.e. show logo, customized text ("Capture your memories with Reecap!") etc..? Is that possible?

**Comment by selbekk** [comment]
Yep - what you're talking about is called the Open Graph project - and what we need is meta data. No problem at all to implement - but I need a few questions answered before I do:

Should the image be the logo? A random picture from the event? A different version of our logo (perhaps just an R or whatever)?

What should the heading read? "[event-name] - Reecap"?
What should the text read? We no longer have a description field to fill out - but there could definitely be a quick reecap sales pitch in there. [Here's all you need to know about the OGP project and what it can do](http://ogp.me/).

I guess support for Twitter cards etc won't be a part of at least this task.

**Comment by chrisbolum** [comment]
Cool!

1) Until we find another version of the Reecap logo, we can use the regular, with blue background and white text. Maybe have rounded edges, so it looks more like an app logo?

2) Good suggestion! Let's use it: "[event-name] - Reecap"

3) It should be a standard pitch for the guests to see: "Upload your memories to Reecap."

We focus only on FB for now :)

**Comment by selbekk** [comment]
You actually have 110 characters to describe reecap - perhaps we can add a short introduction about what it is also? Or why the user that reads this shared post should go in there.

Like, most people that share an event will probably do it with an active event. Therefore, perhaps the text should be something more like "See and share photos from "[event name]" with Reecap. No app required!" - basically write a text that makes sense in the context of reading the shared post.

**Comment by chrisbolum** [comment]
Okay, then it's a good idea to formulate it like that!
